src/main/java/org/prebid/server/bidder/unruly/UnrulyBidder.java

```diff
@@ -132,7 +131,11 @@ private static MultiMap getHeaders() {
     private static BidType getBidType(String impId, List<Imp> imps) {
         for (Imp imp : imps) {
             if (imp.getId().equals(impId)) {
-                return BidType.video;
+                if (imp.getBanner() != null) {
+                    return BidType.banner;
+                } else if (imp.getVideo() != null) {
+                    return BidType.video;
+                }
             }
         }
         throw new PreBidException(String.format("Failed to find impression %s", impId));
```

Comment:
In go they will proceed with BidType.banner and store an error. Current implementation will fail all bid handling. Please, rewrite the logic to match go implementation.

Reply:
just i don't pay attention at this logic, thx